src/lib/header-bidding/prebid/prebid.ts

```diff
@@ -513,15 +512,17 @@ const initialise = (window: Window, consentState: ConsentState): void => {
 		};
 	}
 
-	if (shouldEnableAnalytics()) {
+	if (shouldEnableAnalytics() && window.guardian.ophan?.pageViewId) {
 		window.pbjs.enableAnalytics([
 			{
 				provider: 'gu',
 				options: {
-					ajaxUrl: window.guardian.config.page.isDev
-						? `//performance-events.code.dev-guardianapis.com/header-bidding`
-						: `//performance-events.guardianapis.com/header-bidding`,
-					pv: window.guardian.ophan?.pageViewId,
+					url:
+						window.guardian.config.page.isDev ||
+						window.location.hostname.includes('localhost')
+							? `//performance-events.code.dev-guardianapis.com/header-bidding`
```

Comment:
Would be nice if we could reuse `headerBiddingAnalyticsUrl` here to avoid having these urls and logic defined in multiple places